Radzen.Blazor/Rendering/DialogContainer.razor

```diff
@@ -175,6 +175,15 @@
         }
     }
 
+    string ContentCssClass
+    {
+        get
+        {
+            var baseCss = "rz-dialog-content";
+            return string.IsNullOrEmpty(Dialog.Options.ContentCssClass) ? baseCss : $"{baseCss} {Dialog.Options.ContentCssClass}";
```

Comment:
In my opinion in the else part of this condition you will get twice "rz-dialog-content" since it's already assigned in the DialogService:
```
ContentCssClass = options != null ? $"rz-dialog-content {options.ContentCssClass}" : "rz-dialog-content",
```

Reply:
It's same as others CSS params but I will check that